scalableminds/webknossos
# add optional width and height params to thumbnail api. increase default to 400x400

### Mailable description of changes:
 - Use higher-resolution data for gallery thumbnails

### URL of deployed dev instance (used for testing):
- https://thumbnailsize.webknossos.xyz

### Steps to test:
- thumbnails in gallery should look sharper than before (400×400 rather than 200×200)
- thumbnails should still not feature black borders (or acceptably little) – if that does not work, we might have to change the parameter selection
- try optional url parameters in `/api/datasets/:dataSetName/layers/:layer/thumbnail?w=<Int>&h=<Int>`

### Issues:
- contributes to #2606

------
